test/CMakeLists.txt

```diff
@@ -15,14 +15,15 @@ add_library(valhalla_test
   ${VALHALLA_SOURCE_DIR}/third_party/microtar/src/microtar.h
   ${VALHALLA_SOURCE_DIR}/third_party/microtar/src/microtar.c
   )
-target_include_directories(valhalla_test PUBLIC
-  ${VALHALLA_SOURCE_DIR}/test
-  ${VALHALLA_SOURCE_DIR}/test/gurka
-  ${VALHALLA_SOURCE_DIR}/third_party/protozero/include
-  ${VALHALLA_SOURCE_DIR}/third_party/just_gtfs/include
-  ${VALHALLA_SOURCE_DIR}/third_party/libosmium/include
-  ${robinhoodhashing_include_dir}
-  ${VALHALLA_SOURCE_DIR}/third_party/microtar/src)
+target_include_directories(valhalla_test 
+  PUBLIC
+    valhalla
```

Comment:
instead of writing out any dependencies which are shared with the `valhalla` target, we can just get all of them like this

the below third_party modules are test-specific

Reply:
or not :sweat_smile: